website/search/share_search.py

```diff
@@ -170,8 +173,10 @@ def stats(query=None):
         }
     }
 
-    results = share_es.search(index='share_v1', body=query)
-    date_results = share_es.search(index='share_v1', body=date_histogram_query)
+    results = share_es.search(index=settings.SHARE_ELASTIC_INDEX,
```

Comment:
This actually needs to be ```share_v1``` for the time being, the aggregations are being done on an old schema so that we don't have to update the frontend yet.